tree.go

```diff
@@ -374,13 +374,20 @@ func (roID RosterID) IsNil() bool {
 // RosterTypeID of Roster message as registered in network
 var RosterTypeID = network.RegisterMessage(Roster{})
 
-// NewRoster creates a new ServerIdentity from a list of entities. It also
+// NewRoster creates a new roster from a list of entities. It also
 // adds a UUID which is randomly chosen.
 func NewRoster(ids []*network.ServerIdentity) *Roster {
+	// Don't allow a crash if things are not as expected.
+	if len(ids) < 0 || ids[0].Public == nil {
```

Comment:
@jeffallen sorry, didn't notice this when I first reviewed it. You want `len(ids) < 1`, right? Can you change it in a future commit?

Reply:
Fixing with issue274.